pkgs/tools/text/ocrmypdf/default.nix

```diff
@@ -70,8 +70,8 @@ in buildPythonApplication rec {
 
   postPatch = ''
     substituteInPlace src/ocrmypdf/leptonica.py \
```

Comment:
@dtzWill can this be a patch adding a placeholder (similar to here https://github.com/NixOS/nixpkgs/pull/77527/files), and a `substituteAll` to make it more resilient?

Reply:
@dtzWill poke ;-)

Reply:
Sorry for delay! Good idea, should be good now.  And picked up another version bump while at it :).

Reply:
@flokli Look good? :)

Reply:
Thank you so much for doing this! It would have taken me a long time to figure it out. I tested it earlier and it worked. (I don't think you've changed it since then I think it was last night) The only thing I would change is I prefer that packages use what nixpkgs-fmt says but that's extra work for you and I don't mind doing it the next time I update it. (in general I wish everyone would start doing it too it helps a lot! so maybe consider it in other packages too if you don't do it already ;D )